photoprism/docker-compose.yml

```
@@ -5,6 +5,7 @@ services:
     environment:
       APP_HOST: $APP_PHOTOPRISM_IP
       APP_PORT: $APP_PHOTOPRISM_PORT
+      PROXY_AUTH_WHITELIST: "/originals/*,/import/*"
```

These endpoints have been [confirmed by the developers of PhotoSync](https://mobile.twitter.com/photosync_app/status/1542509638477680642) (which is the [recommended app for uploading to PhotoPrism](https://docs.photoprism.app/user-guide/sync/mobile-devices/)).

---

WYT about documenting _why_ these are whitelisted within this file?

```suggestion
      # These endpoints are protected by HTTP Basic Auth by Photoprism.
      # More details here: https://docs.photoprism.app/user-guide/sync/webdav/#server-url
      PROXY_AUTH_WHITELIST: "/originals/*,/import/*"
```

---

Great suggestion, thanks @tsnieman!